setup.py

```diff
@@ -46,6 +46,23 @@
     install_npm(js_dir, npm=['npm'], build_cmd='build'), ensure_targets(jstargets),
 )
 
+
+extras_require ={
+    'testing': [
+        'pytest>=6.2.4',
+        'anndata==0.7.8',
+        'loompy>=3.0.6',
+    ],
+    'docs': [
+        'sphinx==4.2.0',
+        'sphinx-rtd-theme==1.0.0',
+        'nbclean==0.3.2',
+        'nbsphinx==0.2.13',
+    ],
+}
+
+extras_require['dev'] = ['jupyter_server==1.11.0'] + extras_require['testing'] + extras_require['docs']
```

Comment:
This can be done somewhat more elegantly in YAML within `setup.cfg`. Example,

https://github.com/napari/napari/blob/972b08f8bf26060886cedea57265c37ac9da5d02/setup.cfg#L86-L164

Reply:
The python format is fine with me